composer.json

```diff
@@ -40,30 +43,62 @@
         "sensio/framework-extra-bundle": "^3.0",
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "sulu/document-manager": "~0.10.0",
-        "symfony-cmf/routing-bundle": "^1.2 || ^2.0",
-        "symfony/symfony": "^2.8.13 || ^3.0",
+        "symfony-cmf/routing-bundle": "^2.0",
+        "symfony/asset": "^3.0",
+        "symfony/cache": "^3.0",
+        "symfony/config": "^3.0",
+        "symfony/console": "^3.0",
+        "symfony/dependency-injection": "^3.0",
+        "symfony/doctrine-bridge": "^3.0",
+        "symfony/event-dispatcher": "^3.0",
+        "symfony/expression-language": "^3.0",
+        "symfony/filesystem": "^3.0",
+        "symfony/finder": "^3.0",
+        "symfony/form": "^3.0",
+        "symfony/framework-bundle": "^3.0",
+        "symfony/http-foundation": "^3.0",
+        "symfony/http-kernel": "^3.0",
+        "symfony/intl": "^3.0",
+        "symfony/options-resolver": "^3.0",
+        "symfony/polyfill": "^1.6",
+        "symfony/property-access": "^3.0",
+        "symfony/routing": "^3.0",
+        "symfony/security-bundle": "^3.0",
         "symfony/swiftmailer-bundle": "^3.0",
+        "symfony/templating": "^3.0",
+        "symfony/translation": "^3.0",
+        "symfony/twig-bridge": "^3.0",
+        "symfony/twig-bundle": "^3.0",
+        "symfony/validator": "^3.0",
+        "symfony/web-server-bundle": "^3.0",
         "twig/twig": "^1.11 || ^2.0",
         "willdurand/hateoas-bundle": "^1.1"
     },
     "require-dev": {
-        "phpcr/phpcr-shell": "~1.0",
+        "doctrine/doctrine-fixtures-bundle": "~2.3",
         "jackalope/jackalope-doctrine-dbal": "^1.2.5",
         "massive/build-bundle": "~0.3.0",
         "matthiasnoback/symfony-config-test": "^1.1",
         "matthiasnoback/symfony-dependency-injection-test": "^0.7.3",
-        "phpunit/phpunit": "^4.8.31",
+        "phpcr/phpcr-shell": "~1.0",
+        "phpdocumentor/reflection-docblock": "~3.1",
         "phpspec/prophecy": "^1.4",
-        "symfony/phpunit-bridge": "^2.8.7 || ^3.0",
+        "phpunit/phpunit": "^4.8.31",
         "sensio/framework-extra-bundle": "~3.0",
-        "symfony/monolog-bundle": "^2.8.7 || ^3.0",
+        "symfony/browser-kit": "^3.0",
+        "symfony/css-selector": "^3.0",
+        "symfony/debug": "^3.0",
+        "symfony/dom-crawler": "^3.0",
+        "symfony/debug-bundle": "^3.0",
+        "symfony/monolog-bundle": "^3.0",
+        "symfony/phpunit-bridge": "^3.0",
+        "symfony/process": "^3.0",
+        "symfony/stopwatch": "^3.0",
+        "symfony/var-dumper": "^3.0",
         "zendframework/zend-stdlib": "~2.3",
-        "zendframework/zendsearch": "@dev",
-        "doctrine/doctrine-fixtures-bundle": "~2.3",
-        "phpdocumentor/reflection-docblock": "~3.1"
+        "zendframework/zendsearch": "@dev"
     },
     "conflict": {
-        "symfony/symfony": "2.8.10",
         "symfony/phpunit-bridge": "2.8.10",
         "symfony/monolog-bundle": "2.8.10",
```

Comment:
The above two conflicts could also be removed, right? They are already not possible anymore with the above constraints.